contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -1251,6 +1272,58 @@ SendColumnMetadataToken(int natts, bool sendRowStat)
 			}
 		}
 
+		/* For Spatial Types */
+		tup = SearchSysCache1(TYPEOID, ObjectIdGetDatum(col->pgTypeOid));  /* Search SysCache for type name */
+		typTup = (Form_pg_type) GETSTRUCT(tup);
+
+		/*
+			* Check if it is spatial Data type
+			* Send the Corresponding MetaData Columns
+		*/			
+		if((*common_utility_plugin_ptr -> is_tsql_geometry_datatype) (typTup -> oid) || (*common_utility_plugin_ptr -> is_tsql_geography_datatype) (typTup -> oid))
```

Comment:
nit: what i meant was, should be 

```
if (...)
```

rather than 

```
if(...)
```

and remove the spaces. it should be `foo->bar` not `foo -> bar`. other than that, looks good

Reply:
Miss Understood it, Fixed now